CONTRIBUTING.md

```diff
@@ -24,6 +24,23 @@ Check out our [README.md] for more details.
 
     * The [InnerSource Commons Slack] [#ispo-working-group] community will post the next meeting.
 
+## Our Artifacts
+
+In addition to our [Kanban board] cards, we produce the following artifacts.
+
+```mermaid
+graph LR
+    WG[ISPO Working Group] --> GD[GitHub Discussions]
+    WG --> IP[InnerSource Patterns]
+    WG --> MI[Managing InnerSource Projects]
+    WG --> DR[Google Drive Docs]
+```
+
+[GitHub Discussions] — A place to collect ideas for .
+[InnerSource Patterns CONTRIBUTING.md] — General guidelines for customers of an ISPO.
```

Comment:
I don't understand the "customers of an ISPO" here. Can you elaborate?

Reply:
Yes.

People that benefit from an ISPO's presence at a company.

Reply:
Here's the raw definition on Google that I have in mind:

a person or thing of a specified kind that one has to deal with.

Reply:
I think I am missing context here that would help me understand this. 

What confuses me is the link to the Patterns Contributing.md, which is meant to help users of the ISC patterns to figure out how to contribute, combined with the mention of the customers of an ISPO.

Maybe it is best if I let @rrrutledge do the review of this PR?

Reply:
> Maybe it is best if I let @rrrutledge do the review of this PR?

If you're confused, other people will be confused.

Does this make it more clear?

[InnerSource Patterns] — Contribute to the patterns to improve general guidelines for InnerSource Practitioners.

Reply:
Yes, this is now perfectly clear for me.